common_benchmark_suite/openxla/benchmark/models/pt/resnet/resnet_model.py

```diff
@@ -73,6 +76,8 @@ def postprocess(self, outputs: Tuple[Any, ...]) -> Tuple[Any, ...]:
 def create_model(batch_size: int = 1,
                  data_type: str = "fp32",
                  model_name: str = "torchvision/resnet50",
+                 import_on_gpu: bool = False,
+                 import_with_fx: bool = True,
```

Comment:
The method comment below also needs to be updated.

Reply:
Updated